src/Process/SubdivisionControl.js

```diff
@@ -4,9 +4,25 @@ export default {
             (l, a) => a && a.id == layer.id && l.type == 'color');
         context.elevationLayers = context.view.getLayers(
             (l, a) => a && a.id == layer.id && l.type == 'elevation');
-    },
 
+        context.forceElevation = [];
+        for (const e of context.elevationLayers) {
+            if (e.updateStrategy.type == 1 && e.updateStrategy.options.forceLoadingStep && e.updateStrategy.options.groups) {
+                context.forceElevation = context.forceElevation.concat(e.updateStrategy.options.groups);
+            }
+        }
+        context.forceElevation.sort((a, b) => a - b);
+    },
     hasEnoughTexturesToSubdivide: (context, layer, node) => {
+        if (context.forceElevation.length) {
```

Comment:
Can you document what are the next 5 lines doing, please?

Reply:
My guess is: you prevent subdivision of a node if its elevation level is < to max(valid group levels <= node.level)?

So you need to explain what you do and why you need to do it (maybe with an example?)